contrib/mesos/pkg/executor/executor_test.go

```diff
@@ -437,9 +437,9 @@ func TestExecutorStaticPods(t *testing.T) {
 		assert.NoError(t, err)
 		spod := `{
 	"apiVersion": "v1beta3",
-	"name": "%v",
 	"kind": "Pod",
 	"metadata": {
+		"name": "%v",
```

Comment:
@jdef FYI I had to fix this error in Pod specification. By removing the name defaulting in pkg/kubelet/config/common.go, this test started failing.